Modules/readline.c

```diff
@@ -45,7 +45,6 @@ extern char **completion_matches(char *, CPFunction *);
 #endif
 #endif
 
-#ifdef __APPLE__
 /*
  * It is possible to link the readline module to the readline
  * emulation library of editline/libedit.
```

Comment:
I guess you also ought to remove the "On OSX" part from the comment below (line 52, but github won't let me create a comment there).

Reply:
@labath done.